### PR TITLE
Patterns: use wordpress icons

### PR DESCRIPTION
**Pull request by nightnei**
## Proposed Changes
With this PR we are proposing to use existed WordPress icons instead of icons from the design.
Discussion: p1710182935822239-slack-CNXKTTYCV

<table>
<tr>
<td> From the design
<td> From WordPress icons
</tr>

<tr>
<td> 
<td> 

</tr>

<tr>
<td> 
<td> 
</tr>

<tr>
<td> 
<td> 
</tr>

</table>

### CODE REVIEW

**Comment by fredrikekelund** [comment]
Should I review this one, @nightnei?

File: client/my-sites/patterns/pages/category/index.tsx [hunk 1]
```diff
@@ -106,13 +110,13 @@ export const PatternsCategoryPage = ( {
 					<ToggleGroupControl label="" isBlock className="patterns__toggle-view" value="patterns">
 						<ToggleGroupControlOption
 							value="patterns"
-							label="List view"
-							className="patterns-button__list-view"
+							label={ ( <Icon icon={ iconCategory } size={ 20 } /> ) as unknown as string }
```
Comment (fredrikekelund) [inline client/my-sites/patterns/pages/category/index.tsx]: Wow, that's annoying 😅 I see why you did it, though. Let's keep it as is.

File: client/components/category-pill-navigation/index.tsx [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
+import { Icon, chevronRight } from '@wordpress/icons';
 import classnames from 'classnames';
 import { LocalizedLink } from 'calypso/my-sites/patterns/components/localized-link';
 
 import './style.scss';
 
 type CategoryPillNavigationProps = {
 	buttons?: {
-		icon: string;
+		icon: JSX.Element;
```
Comment (fredrikekelund) [inline client/components/category-pill-navigation/index.tsx]:
```suggestion
		icon: React.ReactElement< typeof Icon >;
```
I guess we could maybe be more specific here if we wanted..?

**Review by fredrikekelund — APPROVED** [review]
LGTM 👍 Left one suggestion, which I'll leave up to you. Other than that, we just need to resolve the merge conflicts before landing

File: client/my-sites/patterns/pages/category/index.tsx (hunk 1 above)
Comment (nightnei) [reply client/my-sites/patterns/pages/category/index.tsx]: Yeah, I was "dancing around" 😅 and it's only one good way :(

**Pull request merged**